Formula/omniorb.rb

```diff
@@ -30,6 +30,7 @@ class Omniorb < Formula
   end
 
   def install
+    ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
```

Comment:
not sure if this is still OK:

```
system "./configure", "--prefix=#{prefix}", "PYTHON=python3"
```

**IMOH**: this is not longer needed

Reply:
You can just make it `python3` too. Better than hardcoding `3.9` where it could get forgotten when the `depends_on "python@3.9"` is changed to something else.